server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala

```diff
@@ -288,7 +288,7 @@ object InteractiveSession extends Logging {
         case (Some(file), false) =>
           debug("Enable HiveContext because hive-site.xml is found under classpath, "
             + file.getAbsolutePath)
-          mergeConfList(List(file.getAbsolutePath), LivyConf.SPARK_FILES)
+          mergeConfList(List(file.getAbsolutePath), LivyConf.SPARK_JARS)
```

Comment:
Hmmm, I think your fix is not correct. It should be added spark.files rather than spark.jars, since this is a file, not a jar.